tests/Twig/Tests/CompletenessTest.php

```diff
@@ -14,8 +14,10 @@
  *
  * @internal
  */
-class CompletenessTest extends \PHPUnit_Framework_TestCase
+final class CompletenessTest extends \PHPUnit_Framework_TestCase
```

Comment:
we've no BC promise on the utest of the project itself, it would be both impractical and useless. We support the API only.